repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/guidance-content.ftl

```diff
@@ -57,6 +79,9 @@
                                             <#case "uk.nhs.hee.web.beans.TabsReference">
                                                 <@hee.tabs tabs=block/>
                                                 <#break>
+                                            <#case "uk.nhs.hee.web.beans.ContentCards">
```

Comment:
Since, this block essentially replaces existing relatedContent cards section, should it need to render full width at the end of the page (assuming Navigation cards content block will be added by Editors as the last item) ?

Reply:
will leave a note for team for consideration. won't fix just yet.